src/controllers/expenses.controllers.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+const expensesServices = require('../services/expenses.services');
+const usersServices = require('../services/users.services');
+
+const getFilteredExpenses = (request, response) => {
+  const expenses = expensesServices.getFiltered(request.query);
+
+  response.send(expenses);
+};
+
+const getOneExpense = (request, response) => {
+  const { expenseId } = request.params;
+
+  if (!expenseId) {
+    response.sendStatus(400);
+
+    return;
+  }
+
+  const foundExpense = expensesServices.getExpenseById(expenseId);
+
+  if (!foundExpense) {
+    response.sendStatus(404);
+
+    return;
+  }
+
+  response.send(foundExpense);
+};
+
+const createExpense = (request, response) => {
+  const {
+    userId,
+    title,
+    spentAt,
+    amount,
+    category,
+  } = request.body;
+
+  if (!userId || !title || !spentAt || !amount || !category) {
+    response.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = usersServices.getUserById(userId);
+
+  if (!foundUser) {
+    response.sendStatus(400);
+
+    return;
+  }
```

Comment:
You want to have more informative error messages - this improves the code readability and maintainability